CHANGELOG.md

```diff
@@ -157,6 +157,7 @@
 
 ### Version "0.5.0"
 
+- Update http crate to 1.0
```

Comment:
Can you move this out of v0.5.0? That version was released so this should go in the next one

Reply:
Of course

Reply:
I moved to 0.6.0, because changing http to 1.0 is a breaking change.

PS: Happy Holidays everyone and have a awesome time family and friends.